src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -78,34 +93,166 @@ public void robotInit() {
     zeroElevator.setRunWhenDisabled(true);
     SmartDashboard.putData(zeroElevator);
 
-    UsbCamera camera = CameraServer.getInstance().startAutomaticCapture(Tuning.camID);
-    camera.setResolution(128, 73);
-    camera.setFPS(30);
+    Command zeroDrivetrain = new SimpleCommand("[Drivetrain] Zero Drivetrain", drivetrain::zeroEncoders);
+    zeroDrivetrain.setRunWhenDisabled(true);
+    SmartDashboard.putData(zeroDrivetrain);
+
+    UsbCamera overheadCam = CameraServer.getInstance().startAutomaticCapture(Tuning.overheadCamID);
+    overheadCam.setResolution(128, 73);
+    overheadCam.setFPS(30);
+
+    UsbCamera turretCam = CameraServer.getInstance().startAutomaticCapture(Tuning.turretCamID);
+    turretCam.setResolution(128, 73);
+    overheadCam.setFPS(30);
+
+
+    Command refreshProfiles = new SimpleCommand("[MotionP] Refresh Motion Profiles",
+        () -> profiles = new CSVProfileManager(new File("/home/lvuser/profiles")));
+    refreshProfiles.setRunWhenDisabled(true);
+    SmartDashboard.putData(refreshProfiles);
 
     // initialize profiles
     // unlike other static fields, initialized here because there's a high likelihood of it throwing
     // an exception and exceptions thrown during static initialization are not fun.
     profiles = new CSVProfileManager(new File("/home/lvuser/profiles"));
+
+    SmartDashboard.putData(drivetrain);
+    SmartDashboard.putData(PowerManager.getInstance());
   }
 
   @Override
   public void disabledInit() {
+    Robot.drivetrain.configTalonsForVelocity();
+    if (autoCommand != null) {
+      autoCommand.cancel();
+    }
   }
 
+  @SuppressWarnings("Duplicates")
   @Override
   public void autonomousInit() {
```

Comment:
Surely this could be made way, way, way nicer using enums and methods around those?

Reply:
yes if you want to do that be my guest

Reply:
Will do later this week, I have lots of HW.